### ISSUE TITLE
Artifact removal average

### ISSUE THREAD

**Issue by CGonzC** [SpikeInterface/spikeinterface#2960]
Hello!

I am trying to remove the light-induced artifact in my recordings with the remove_artifacts function. It works perfectly when "cubic" or "linear" are used. When I try to use the "average" method, it apparently does the average computations, but no changes are observed in the recordings.
![image](https://github.com/SpikeInterface/spikeinterface/assets/155523434/bb7ce36c-373d-4455-8c73-21b430f58084)

What am I missing?

Thank you very much in advance.

Cristian

**Comment by CGonzC** [comment]
HI @alejoe91 ,

Thanks for your quick reply!
They are very similar artifacts per channel. They differ among channels only. Regarding the alignment, they are bound to the light TTL, so I expect no changes there.
 What I get is an unchanged recording, the artifacts are identical when superimposed with the original recording. 
Should I change anything else in the function out of the default?

Thanks again!

Cristian

**Comment by alejoe91** [comment]
Mmm that is strange. What version are you using?

@yger any idea what could be wrong?

**Comment by CGonzC** [comment]
I'm using SpikeInterface version: 0.100.0.dev0

**Comment by alejoe91** [comment]
Can you try to plot the content of:

```
rec_cleaned4._kwargs["artifacts"]
```
This are the estimated average artifacts

**Comment by CGonzC** [comment]
It worked after updating SI to version 0.101.0. 

@alejoe91  I am so sorry for wasting your time.

Thank you very much!!!
Cristian

**Comment by alejoe91** [comment]
No worries at all! Closing then :)

**Comment by CGonzC** [comment]
Hi @alejoe91 ,

after the update I got a problem in the sorting part. I am running kilosort3 in Docker . But now I got an error:
TypeError: unsupported operand type(s) for /: 'NoneType' and 'str'

![image](https://github.com/SpikeInterface/spikeinterface/assets/155523434/15d67879-c5f9-4932-b5cb-fab004aca265)

Before the update all worked perfectly.

What could be the problem?

Thank you very much in advance

Cristian
